.github/workflows/netlify.yaml

```diff
@@ -0,0 +1,49 @@
+name: netlify production
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node: [10.x, 12.x, 14x]
```

Comment:
I don't think we need a node matrix for this. Simply node version 12 is sufficient